src/player/QtHelper.h

```diff
@@ -196,8 +196,7 @@ struct node_builder {
             mpv_node_list *list = dst->u.list;
             if (list) {
                 for (int n = 0; n < list->num; n++) {
-                    if (list->keys)
-                        delete[] list->keys[n];
+                    delete[] list->keys[n];
```

Comment:
:thought_balloon: This adjustment might need [further explanations](https://github.com/mpv-player/mpv/blob/c0d989c83913fb9c990d961c44da4816b8d2de18/include/mpv/client.h#L803-L830 "Background information").